src/status_im2/subs/communities.cljs

```diff
@@ -105,6 +105,28 @@
  (fn [communities]
    (map :id communities)))
 
+
+;; Return communities splitted by level of user participation. Some communities user
+;; already joined, to some of them join request sent and others were opened one day
+;; and their data remained in app-db.
+;; Result map has form: {:joined [id1, id2] :pending [id3, id5] :opened [id4]}"
+
+
+(re-frame/reg-sub
+ :communities/community-ids-by-user-involvement
```

Comment:
How about tests for this new layer-3 subscription? It's begging for some! If it's a new layer-3 subscription such as this one, we should properly test it. Feel free to ping me if it's not clear how to test this. You can find examples in:

- subs/communities_test.cljs
- subs/wallet/wallet_test.cljs
- subs/activity_center_test.cljs

On a side note, I actually think soon we should re-evaluate our minimum requirements for test coverage in PRs.